knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Modifying GridView Exported XLSX Spreadsheet
+description: How to make cusomt changes to exported spreadsheet document when exporting GridView to Excel xlsx.
```

Comment:
- Instead of "How to", consider starting with a verb.  
- *custom

Reply:
Also caught a misspelling there (custom)